snippets/base-class.js

```diff
@@ -0,0 +1,82 @@
+/* We create a new class that extends from the native HTMLElement */
+class BatchingElement extends HTMLElement {
+  constructor() {
+
+    /* We now have to call `super` to make sure our element gets set up correctly */
```

Comment:
```suggestion
```
I think the super call is relevant to a beginner wc post, but this one assumes some familiarity?